### DISCUSSION TITLE
alist挂载了阿里云盘使用302重定向，在局域网内可以满速下载，但是使用公网下载就只能走上传的带宽

### DISCUSSION THREAD

**Discussion by Julian-Won** [AlistGo/alist#6009] [General]
在群晖docker container中搭建好了alist，使用阿里云盘Open进行挂载，挂载方式是302：
![image](https://github.com/alist-org/alist/assets/67736924/124613dc-a802-43d1-a436-9a9e485e2a29)

本地成功访问alist，并且能够下载视频
![image](https://github.com/alist-org/alist/assets/67736924/d4715253-cec9-4409-adaf-b1faa44d6cd0)
可以看到速度是满速的。

我自己是电信网络，所以使用旁路由进行了ddns进行绑定域名，在外面使用公网访问的时候，发现流量全部走了alist挂载服务器的上传流量上：
![image](https://github.com/alist-org/alist/assets/67736924/1ec3585d-72b0-47fc-97fd-a36670f3b4fd)
我想问重定向302只对局域网有作用吗，因为家里开了pcdn的服务，开启了upnp功能，是不是这个影响了呢，但是我自己搭建的小雅alist下载视频的速度还是很可观的：
小雅alist下载视频的速度：（与alist一样放在群晖docker中）
![image](https://github.com/alist-org/alist/assets/67736924/3e58ec43-631f-42e8-a248-f5997be944c7)

**Comment by xhofe** [comment]
https://alist.nn.ci/zh/guide/drivers/common.html#web-%E4%BB%A3%E7%90%86

**Reply by Julian-Won** [reply]
我根据文档设置了，请问是我没有看到什么配置吗

**Reply by Julian-Won** [reply]
谢谢，我已经解决了，关闭web代理，只开302定向就可以了，谢谢
